src/pages/media.js

```diff
@@ -16,7 +16,7 @@ export default () => {
     return (
         <Layout>
             {(isLoading || isLoadingPodcasts) && <P>Loading...</P>}
-            {videos && <CardList videos={videos} podcasts={podcasts} />}
+            <CardList videos={videos} podcasts={podcasts} />
```

Comment:
This line was part of the problem, if we had videos first but not podcasts, podcasts would pass a value `null` which would cause issues in the cardlist. I moved this type of checking to the cardlist so callsites don't need to worry about input validation.